crates/trippy-tui/locales/app.yml

```diff
@@ -6,730 +6,835 @@ trippy:
+  ru: "Цвета темы Tui"
 settings_tab_columns_desc:
   en: "Tui table columns.  Press [%{c}] to toggle a column on or off and use the [%{d}] and [%{u}] keys to change the column order."
   fr: "Colonnes de table Tui. Appuyez sur [%{c}] pour activer ou désactiver une colonne et utilisez les touches [%{d}] et [%{u}] pour changer l'ordre des colonnes."
   tr: "Tui tablo sütunları. Bir sütunu açmak veya kapatmak için [%{c}] tuşuna basın ve sütun sırasını değiştirmek için [%{d}] ve [%{u}] tuşlarını kullanın."
   it: "Colonne della tabella Tui. Premi [%{c}] per attivare o disattivare una colonna e usa i tasti [%{d}] e [%{u}] per cambiare l'ordine delle colonne."
   pt: "Colunas da tabela Tui. Pressione [%{c}] para ativar ou desativar uma coluna e use as teclas [%{d}] e [%{u}] para alterar a ordem das colunas."
   zh: "终端用户界面表格列。按 [%{c}] 切换列的显示和隐藏，使用 [%{d}] 和 [%{u}] 键更改列的顺序。"
+  ru: "Столбцы таблицы Tui. Нажмите [%{c}], чтобы включить или отключить столбец, и используйте клавиши [%{d}] и [%{u}], чтобы изменить порядок столбцов."
 settings_table_header_setting:
   en: "Setting"
   fr: "Paramètres"
   tr: "Ayar"
   it: "Impostazione"
   pt: "Configuração"
   zh: "设置"
+  ru: "Настройка"
 settings_table_header_value:
   en: "Value"
   fr: "Valeur"
   tr: "Değer"
   it: "Valore"
   pt: "Valor"
   zh: "值"
+  ru: "Значение"
 column_host:
   en: "Host"
   fr: "Hôte"
   tr: "Ana bilgisayar"
   it: "Host"
   pt: "Host"
   zh: "主机"
+  ru: "Хост"
 column_loss_pct:
   en: "Loss%"
   fr: "% Perdus"
   tr: "Kayıp%"
   it: "Persi%"
   pt: "% Perdidos"
   zh: "丢包率"
+  ru: "Потери%"
 column_snd:
   en: "Snd"
   fr: "Envoyés"
   tr: "Gönderilen"
   it: "Snd"
   pt: "Enviados"
   zh: "发出"
+  ru: "Отпр"
 column_recv:
   en: "Recv"
   fr: "Reçus"
   tr: "Alınan"
   it: "Recv"
   pt: "Recebidos"
   zh: "接收"
+  ru: "Получ"
 column_last:
   en: "Last"
   fr: "Dernier"
   tr: "Son"
   it: "Ultimo"
   pt: "Último"
   zh: "最后"
+  ru: "Посл"
 column_avg:
   en: "Avg"
   fr: "Moyenne"
   tr: "Ort"
   it: "Media"
   pt: "Média"
   zh: "平均"
+  ru: "Ср"
 column_best:
   en: "Best"
   fr: "Meilleur"
   tr: "En iyi"
   it: "Migliore"
   pt: "Melhor"
   zh: "最佳"
+  ru: "Луч"
 column_wrst:
   en: "Wrst"
   fr: "Pire"
   tr: "En kötü"
   it: "Peggiore"
   pt: "Pior"
   zh: "最差"
+  ru: "Худ"
 column_stdev:
   en: "StDev"
   fr: "ÉcTyp"
   tr: "StDev"
   it: "StDev"
   pt: "DesvPad"
   zh: "标准差"
+  ru: "СКО"
 column_sts:
   en: "Sts"
   fr: "État"
   tr: "Sts"
   it: "Stato"
   pt: "Est"
   zh: "状态"
+  ru: "Статус"
 column_jttr:
   en: "Jttr"
   fr: "Gigue"
   tr: "Jttr"
   it: "Jttr"
   pt: "Jttr"
   zh: "抖动"
+  ru: "Джитр"
 column_javg:
   en: "Javg"
   fr: "GigMoy"
   tr: "Javg"
   it: "Javg"
   pt: "Javg"
   zh: "均抖"
+  ru: "СреднДжитр"
 column_jmax:
   en: "Jmax"
   fr: "GigMax"
   tr: "Jmax"
   it: "Jmax"
   pt: "Jmax"
   zh: "最大抖"
+  ru: "МаксДжитр"
 column_jint:
   en: "Jint"
   fr: "GigInt"
   tr: "Jint"
   it: "Jint"
   pt: "Jint"
   zh: "抖动间隔"
+  ru: "ИнтДжитр"
 column_sprt:
```

Comment:
Not clear context. What's sprt/dprt here?

Reply:
Source and destination ports. Keep in mind these are column names and horizontal space is at a premium so short/abbreviated words are best.